Tests/GeoLib/TestLineSegmentIntersect2d.cpp

```diff
@@ -50,18 +50,29 @@ class LineSegmentIntersect2dTest : public testing::Test
 	ac::gtest_reporter gtest_reporter;
 };
 
+#if not defined(_MSC_VER) && _MSC_VER < 1900
+// Compilers of MVS below 2015 do not support unrestricted unions. The
+// unrestricted union is used by autocheck to handle test data. The autocheck
+// workaround for MVS compilers (below version 2015) contains a bug and in the
+// consequence the tests crashes. For this reason the tests are disabled under
+// this environments.
```

Comment:
I checked this. The result is that the code doesn't compile when unrestricted unions aren't supported because then a default constructor is needed. But I don't want to implement the default constructor only for the reason that *disabled* tests can be compiled.